testing/features/step_definitions/breadcrumbs_steps.rb

```diff
@@ -3,6 +3,7 @@
 Given("a Default Breadcrumbs component is on the page") do
   @component = Breadcrumbs::Default.new
   @component.load
+  @default = true
```

Comment:
This shouldn't be needed. We can either check the class, or do something else. But I'm fairly certain just the mobile_phone? check should be fine.

Reply:
The problem here is not the small_screen of mobile_phone. The problem is in the step "But the final breadcrumb isn't a link", that is not true for breadcrumb_default:
<img width="323" alt="Screenshot 2020-12-16 at 17 38 21" src="https://user-images.githubusercontent.com/11262774/102385389-b3a56400-3fc5-11eb-953d-354ae109771e.png">
So maybe we need to change the step to be like: But the final breadcrumb isn't a link unless on default? What do you think @luke-hill